addons/website_slides/tests/test_slide_channel.py

```diff
@@ -274,6 +275,47 @@ def _get_completion_time_pdf(*args, **kwargs):
         self.assertEqual(13.37, round(slides_1.completion_time, 2))
         self.assertEqual(123.0, slides_2.completion_time)
 
+    def test_if_unlink_and_unpublish_channel_slides_sends_competition_email(self):
+        """When all content (slide.slide) are deleted from the course (slide.channel),
+        the course participants (slide.channel.partner) SHALL NOT receive completers email notification. """
+
+        with self.mock_mail_gateway():
+            _count_send_mails = lambda: len(self._new_mails.filtered(lambda m: m.model == 'slide.channel.partner'))
```

Comment:
It's not good practice to assign lambda functions to variables, you can however make a local function.

Also, if we're always comparing to 0, why not return a boolean instead?
```suggestion
            def _did_send_email_to_attendees():
                return bool(self._new_mails.filtered(lambda m: m.model == 'slide.channel.partner'))
```

Reply:
good point, committing